slices/git.yaml

```diff
@@ -0,0 +1,57 @@
+package: git
+
+essential:
+  - git_copyright
+
+slices:
+  bins:
+    # Basic functionality including:
+    # - add
+    # - blame
+    # - branch
+    # - checkout
+    # - commit
+    # - config
+    # - diff
+    # - fetch
+    # - init
+    # - log
+    # - merge
+    # - rev-list
+    # - status
+    essential:
+      - libc6_libs
+      - libpcre2-8-0_libs
+      - zlib1g_libs
+    contents:
+      /usr/bin/git: {symlink: /usr/lib/git-core/git}
```

Comment:
Let's please preserve this comment from 24.04, since we are creating this symlink ourselves.

```diff
-      # /usr/bin/git is a regular file in the package, but shares
-      # the same hash of /usr/lib/git-core/git. That's why this
-      # is added as a symlink to reduce space.
```

Reply:
```suggestion
      # /usr/bin/git is a regular file in the package, but shares
      # the same hash of /usr/lib/git-core/git. That's why this
      # is added as a symlink to reduce space.
      /usr/bin/git: {symlink: /usr/lib/git-core/git}
```